### PR TITLE
improved error reporting for unhandled errors;

### PR DESCRIPTION
**Pull request by spherefoundry**
# Checklist:

- [x] Code follows the style guidelines
- [x] Documentation updated
- [x] Corner cases evaluated and handled
- [x] Automatic tests added/update
- [x] Automatic tests pass
- [x] Software runs (locally or hosted)
- [x] Sanity check passed
- [x] Self-review passed

### CODE REVIEW

File: src/utils/instrument_process.js [hunk 1]
```diff
@@ -11,8 +11,15 @@ import * as Sentry from '@sentry/node';
...
+const serialiseError = (error) => Object.getOwnPropertyNames(error).reduce(
+  (accum, key) => {
+    accum[key] = error[key];
+    return accum;
+  },
+  {name: error.name});
+
 const reportAndExit = async (sentryClient, logger, message, error) => {
-  logger.error({message, error});
+  logger.error({message, error: serialiseError(error)});
```
Comment (vanruch) [inline src/utils/instrument_process.js]:
```suggestion
  logger.error({message, error: serializeError(error)});
```

File: src/utils/instrument_process.js [hunk 2]
```diff
@@ -11,8 +11,15 @@ import * as Sentry from '@sentry/node';
 import {WinstonConsoleLogger} from './loggers';
 import config from '../../config/config';
 
+const serialiseError = (error) => Object.getOwnPropertyNames(error).reduce(
```
Comment (vanruch) [inline src/utils/instrument_process.js]:
```suggestion
const serializeError = (error) => Object.getOwnPropertyNames(error).reduce(
```
